app/templates/legal/cookiestatement.hbs

```diff
@@ -1,6 +1,6 @@
 {{page-title "Cookieverklaring"}}
 
-<AuContent class="au-u-3-4@medium">
+<AuContent class="au-u-3-4@medium" {{auto-focus}} tabindex="-1">
```

Comment:
I would explicitly install the auto-focus addon in the app. It only works because ember-appuniversum brings it in at the moment.